libs/utils/src/completion.rs

```diff
@@ -49,5 +51,5 @@ pub fn channel() -> (Completion, Barrier) {
     tracker.close();
 
     let token = tracker.token();
-    (Completion(token), Barrier(tracker))
```

Comment:
Why this?

Reply:
Aah, better no one reads the field, interesting.

Reply:
Unsure how I feel about this warning for single field  newtypes. Doing exactly this is a very valid pattern... I would hope they'd check for drop in the lint before stabilizing this.